mkosi/resources/man/mkosi.1.md

```diff
@@ -1046,11 +1046,8 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     the image.
 
 `KernelModulesInitrd=`, `--kernel-modules-initrd=`
-:   Enable/Disable generation of the kernel modules initrd when building a bootable image. Enabled by default.
-    If enabled, when building a bootable image, for each kernel that we assemble a unified kernel image for we
-    generate an extra initrd containing only the kernel modules for that kernel version and append it to the
-    prebuilt initrd. This allows generating kernel independent initrds which are augmented with the necessary
-    kernel modules when the UKI is assembled.
+:   Enable/Disable generation of the kernel modules' initrd when building a bootable image. Enabled by default.
+    If enabled, when building a bootable image, mkosi will generate an extra initrd for each kernel version it assembles. Such an initrd only contains modules to append to the prebuilt initrd. This allows generating kernel independent initrds which are augmented with the necessary modules when the UKI is assembled.
```

Comment:
Please don't put this on a single long line, but reflow to 109 columns like it previously was.

Reply:
The changes to the first and last sentence are an improvement, but the middle one loses some information.
We don't assemble kernels or kernel versions, we assemble UKIs, i.e. jam a kernel image and initrds and some other things into a PE file. Also, the third sentence (split of from the second and somewhat abstractly starting with "such") leaves it a bit too open what modules are now in this initrd:
```suggestion
    If enabled, when building a bootable image, mkosi will generate an extra initrd for each unified kernel image it assembles. This initrd contains only modules for the specific kernel version and will be appended to the prebuilt initrd. This allows generating kernel independent initrds which are augmented with the necessary modules when the UKI is assembled.
```

Reply:
indeed important because in the end, this setting only applies when generating a UKI, working on the suggested corrections.

Reply:
> Please don't put this on a single long line, but reflow to 109 columns like it previously was.

got it, adding to my linting